.eslintrc

```diff
@@ -1,3 +1,8 @@
 {
-  "extends": "eslint-config-fliplet"
+  "extends": "eslint-config-fliplet",
+  "globals": {
+    "TD": "readonly",
```

Comment:
These should not be necessary if you're on the latest version of Fliplet's ESLint config. Please correct this for all other PRs that aren't using the latest ESLint config.